actions/workflows/st2_pkg_upgrade_e2e_test.yaml

```diff
@@ -28,7 +26,10 @@ vars:
   - vm_info:
   - vm_id:
   - vm_windows_id:
-  - windows_hostname: <% ctx().hostname %>win
+  - vm_windows_info:
```

Comment:
Perhaps  comment out windows_hostname rather than delete and a comment to say this is temporary until we re-instate windows hosts.